### PR TITLE
[PM-10045] - SendCreated page

### PR DESCRIPTION
**Pull request by jaasen-livefront**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-10045

## 📔 Objective

This PR adds the send created page as part of the browser refresh. It is shown as a result of a successfully created send which does currently exist so QA testing will not be possible. To enable this page on local development you may replace the `SendV2Component` with the `SendCreatedComponent` in `app-routing.module.ts`.

## 📸 Screenshots

Screenshot 2024-08-13 at 11 35 17 AM

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: apps/browser/src/_locales/en/messages.json [hunk 1]
```diff
@@ -2177,6 +2177,18 @@
     "message": "Send created",
     "description": "'Send' is a noun and the name of a feature called 'Bitwarden Send'. It should not be translated."
   },
+  "createdSendSuccessfully": {
+    "message": "Send created successfully!",
+    "description": "'Send' is a noun and the name of a feature called 'Bitwarden Send'. It should not be translated."
+  },
+  "sendAvailability": {
+    "message": "The Send will be available to anyone with the link for the next [7] days.",
```
Comment (djsmith85) [inline apps/browser/src/_locales/en/messages.json:2185]: The number of days will be dynamic based on the settings (expiration/deletion date) of the Send

File: libs/angular/src/jslib.module.ts [hunk 2]
```diff
@@ -132,6 +132,7 @@ import { IconComponent } from "./vault/components/icon.component";
     FallbackSrcDirective,
     I18nPipe,
     IconComponent,
+    IconModule,
```
Comment (djsmith85) [inline libs/angular/src/jslib.module.ts:135]: This module should be removed and can be added to the imports of the standalone send.created.component

**Review by djsmith85 — CHANGES_REQUESTED** [review]
@Jordan Aasen Nice work. 
We still need to think of a way to pass the saved Send or SendId over to this page. The link and the number of days it's available are going to depend on the settings of the Send

**Comment by jaasen-livefront** [comment]
@djsmith85 Ready for re-review!

**Review by danielleflinn — CHANGES_REQUESTED** [review]
@jaasen-livefront can you remove the brackets around the "7"? 

This punctuation was in Figma just to communicate that this value should dynamically change based on the deletion date the user selects

**Comment by jordan-aasen** [comment]
@danielleflinn Nice catch! Looks like I posted an older screenshot. I've updated it with one that no longer contains the brackets.

**Review by djsmith85 — APPROVED** [review]

**Review by danielleflinn — APPROVED** [review]

**Pull request merged**